### DISCUSSION TITLE
PublicSuffixList API proposal to the WECG

### DISCUSSION THREAD

**Discussion by dannycolin** [uBlockOrigin/uBlock-issues#2151] [General]
Hi @gorhill,

I simply wanted to share that there's been a proposal made (See: https://github.com/w3c/webextensions/issues/231) for a WebExtension API that'd let you call the browser function to interact with the PublicSuffixList. I know that Ublock Origin uses a bundled version of it and thought it may interest you. 

Also if anyone here has interest in this functionality feel free to subscribe to the issue or share your use case as an addon developer.

P.S.: Sorry for using the discussion for that. I couldn't find a way to get in touch with you. 

Best regards,

**Comment by gorhill** [comment]
Cool, thanks for the heads up.
